feathr_project/feathr/registry/_feathr_registry_client.py

```diff
@@ -122,6 +122,8 @@ def register_features(self, workspace_path: Optional[Path] = None, from_context:
         for df in _topological_sort(derived_feature_list):
             if not hasattr(df, "_registry_id"):
                 df._registry_id = self._create_derived_feature(df)
+        url = '/'.join(self.endpoint.split('/')[:3])       
+        logger.info(f"Check project lineage by this link: {url}/projects/{self.project_name}/lineage")
```

Comment:
If user register with direct purview write, which means no endpoint is specified in config, will this code work?

Reply:
For that case, this method will not be called. The 'register_features' method in '_feathr_registry_purview' will be called instead with the given purview name.

Reply:
Just to confirm, do we have a similar log after registering features in the direct purview write mode?